### DISCUSSION TITLE
New line automatically after auto-paste?

### DISCUSSION THREAD

**Discussion by larhyp** [RyotaUshio/obsidian-pdf-plus#135] [Q&A] [answered]
Hi,

What a great plugin this is! Just started using it recently. I'm having 'Toggle auto-copy' and Toggle auto-paste' on, so the highlights will be copied automatically to the markdown file. Question: Is there a way to get a new line (enter) automatically after paste? Otherwise the copied highlights will be cluttered and every time I have to manually enter the new line after the paste.

**Comment by RyotaUshio** [comment, marked as answer]
Do you know the copy formats can be customized as you like?
So you can add a line break after each [copy format](https://ryotaushio.github.io/obsidian-pdf-plus/link-copy-formats.html).

For example, the default "Callout" format is defined as

```
> [!{{calloutType}}|{{color}}] {{linkWithDisplay}}
> {{text}}

```

(You can find this in PDF++ settings > Link copy templates > Custom link copy formats; if you find it tedious to scroll down the settings page, just copy & paste `obsidian://pdf-plus?setting=copyCommands` to your browser's search bar and open this URL)

You can add a new line at the end of this, like so

```
> [!{{calloutType}}|{{color}}] {{linkWithDisplay}}
> {{text}}


```

**Reply by larhyp** [reply]
Thanks, did not know, that works like charm!!
